### DISCUSSION TITLE
ProxSuite 0.2.7

### DISCUSSION THREAD

**Discussion by jcarpent** [Simple-Robotics/proxsuite#102] [Announcements]
## What's Changed
* CMakeLists: fix INTERFACE target compile definitions by @fabinsch in https://github.com/Simple-Robotics/proxsuite/pull/97
* sync submodule by @fabinsch in https://github.com/Simple-Robotics/proxsuite/pull/98
* Fixed temporaries in update() of dense wrapper by @fennel-labs in https://github.com/Simple-Robotics/proxsuite/pull/100
* Additional option to check eigen runtime malloc by @fabinsch in https://github.com/Simple-Robotics/proxsuite/pull/93

## New Contributors
* @fennel-labs made their first contribution in https://github.com/Simple-Robotics/proxsuite/pull/100

**Full Changelog**: https://github.com/Simple-Robotics/proxsuite/compare/v0.2.6...v0.2.7

<hr /><em>This discussion was created from the release <a href='https://github.com/Simple-Robotics/proxsuite/releases/tag/v0.2.7'>ProxSuite 0.2.7</a>.</em>

**Comment by fabinsch** [comment]
Hello, I just compared the performance of ProxQP v0.2.7 to v0.2.5 with our [benchmarks](https://github.com/Simple-Robotics/proxqp_benchmark) and I can see an average speedup of 8% for the `Random Mixed QP` problems. Great work @fennel-labs and @jcarpent to fix the memory allocations. @stephane-caron
